tests/conftest.py

```diff
@@ -179,7 +179,9 @@ def dump(self, result):
             self.__dump_pretty_json(self._results_file, result, "a")
 
 
-def init_microvm(root_path, bin_cloner_path, fc_binary=None, jailer_binary=None):
+def init_microvm(
+    root_path, bin_cloner_path, fc_binary=None, jailer_binary=None, memory_monitor=True
+):
```

Comment:
This conflicts with #3449 . This function goes away there and maybe this could be simpler then?

Reply:
Oh, lets get your PR merged first then, and I'll rebase! :)